client/assets/js/main.js

```diff
@@ -2,42 +2,50 @@
 
 import Board from './board';
 import server from './sendToServer.js';
+import $ from 'jquery';
 
 document.addEventListener('DOMContentLoaded', function () {
 
-	let board = new Board();
+	$('#form-name').submit(function(e) {
+		e.preventDefault();
+		var name = $('#name')[0].value;
 
+		$("#form-name").addClass("filled");
```

Comment:
`$("#form-name")` triggers a DOM query. Create a variable (given the result won't change).
Maybe prefer `"form#name"` given the element cannot be anything else than a form ;-)